src/cart/crt_iv.c

```diff
@@ -2720,6 +2720,7 @@ handle_response_cb_internal(void *arg)
 static void
 handle_response_cb(const struct crt_cb_info *cb_info)
 {
+	void			*cb_info2 = (void *)cb_info;
```

Comment:
You probably don't need a cast here either.

Reply:
This is required because of I loose the CONST. Compiler will complain about this.

Reply:
Ah yes, that's annoying that is.